Makefile

```diff
@@ -29,13 +29,18 @@ checkstyle:
 build: build/distributions/tiered-storage-for-apache-kafka-$(VERSION).tgz
 
 build/distributions/tiered-storage-for-apache-kafka-$(VERSION).tgz:
-	./gradlew build distTar -x integrationTest -x e2e:test
+	./gradlew build distTar -x test -x integrationTest -x e2e:test
```

Comment:
Maybe should be done in this PR, but it seems to me it's worth having a make target that does the full build with tests (e.g. locally).